Source/WTF/wtf/PlatformHave.h

```diff
@@ -1820,3 +1820,40 @@
 #if !defined(HAVE_UITOUCH_ROLLANGLE) && (PLATFORM(IOS) || PLATFORM(MACCATALYST) || PLATFORM(VISION))
 #define HAVE_UITOUCH_ROLLANGLE 1
 #endif
+
+#if !defined(HAVE_WEB_TRANSPORT) \
+    && (((PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 150400) \
+    || ((PLATFORM(IOS) || PLATFORM(MACCATALYST)) && __IPHONE_OS_VERSION_MIN_REQUIRED >= 180400) \
```

Comment:
We didn't need most of this, certainly iOS 18.4 is older than the oldest supported.